contentpool_client.module

```diff
@@ -1,6 +1,82 @@
 <?php
 
+use Drupal\Core\Form\FormStateInterface;
+
 /**
  * @file
  * Module file.
  */
+
+/**
+ * Implements HOOK_form_FORM_ID_alter.
+ */
+function contentpool_client_form_remote_edit_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $remote = $form_state->getFormObject()->getEntity();
+
+  $form['actions']['submit']['#submit'] = array_merge(['contentpool_client_form_remote_edit_form_submit'], $form['actions']['submit']['#submit']);
+
+  $form['contentpool_client_settings'] = [
+    '#type' => 'details',
+    '#title' => t('Contentpool client settings'),
+    '#open' => TRUE,
+  ];
+
+  $form['contentpool_client_settings']['autoregister'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Autoregister this site to remote site (if supported)'),
+    '#description' => t('If the remote site is a contentpool server, the site can be autoregistered remotely.'),
+    '#default_value' => $remote->getThirdPartySetting('contentpool_client', 'autoregister', 0)
+  ];
+
+  $form['contentpool_client_settings']['autopull'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Automatically pull content from remote contentpool server regularly.'),
+    '#description' => t('Regularly triggers a content pull from remote site.'),
+    '#default_value' => $remote->getThirdPartySetting('contentpool_client', 'autopull', 0)
+  ];
+
+  $form['contentpool_client_settings']['autopull_interval'] = [
+    '#type' => 'select',
+    '#title' => t('Automatic pull interval'),
+    '#description' => t('The automatic pull is triggerd by cron. Your cron interval is the lower limit.'),
```

Comment:
Let's support this without cron also, that means:
- We need to provide a drush command that triggers the same logic. I think the drush command should just by-pass the interval checks but respect the autopull flag.

Reply:
Let's use that description:

If set to "Never" the automatic pull has to be triggered manually via the provided drush command. Automatic pulls are triggerd by cron, thus your cron interval is the lower limit.